example-mlem-get-started/code/README.md

```diff
@@ -0,0 +1,44 @@
+# MLEM Get Started
+
+This is an auto-generated repository for use in MLEM
+[Get Started](https://mlem.ai/doc/get-started). It is a step-by-step quick
+introduction into basic MLEM concepts.
+
+- The `main` branch contains only the code needed to start the Get Started. This
```

Comment:
That's quite unusual - downside is that people who come to the repo won't see anything in it. This is not good to my mind. What is the purpose of keeping the main like this at all if we have the corresponding tags? Can it be done this way:

- `simple` becomes `main`
- `dvc` - can it be done in way that builds on top of simple - we can make a PR from `dvc` to `main` (ex-simple) to DVC-ify the project?

Reply:
Did that and committed changes. Please see https://github.com/iterative/mlem.ai/pull/132 for PR updating documentation.